ios/trainer.xcodeproj/project.pbxproj

```diff
@@ -874,7 +874,7 @@
 					00E356ED1AD99517003FC87E = {
 						CreatedOnToolsVersion = 6.2;
 						DevelopmentTeam = 29DWXCY7LN;
-						ProvisioningStyle = Manual;
+						ProvisioningStyle = Automatic;
```

Comment:
Can you give me the quick and dirty on what these changes are?

Reply:
Basically we need to get a provisioning certificate to our app builds. a provisioning certificate basically says "yo this app is able to be installed on this device for blah reason" in this case we're switching I believe the dev build to use the dev provisioning license, and the release build to use the ad hoc provisioning license, which basically allows us to distribute our app to multiple test devices (think beta users), I'll add you to appCenter and you can download a release version of the build, you'll notice it's snappier and you can't develop with it